pkg/cmd/pipeline/delete.go

```diff
@@ -28,7 +27,7 @@ import (
 )
 
 func deleteCommand(p cli.Params) *cobra.Command {
-	opts := &options.DeleteOptions{Resource: "pipeline", ForceDelete: false, DeleteRelated: false}
+	opts := &options.DeleteOptions{Resource: "pipeline", ForceDelete: false, DeleteRelated: false, DeleteAllNs: false}
```

Comment:
Nit: `DeleteAllNs` looks like delete all namespaces

Reply:
It’s currently used in every delete command across the CLI, so I would recommend opening a separate issue on this that would be handled separately than in this pr. 

I do see your point though on it being confusing.

Reply:
It’s currently used in every delete command across the CLI, so I would recommend opening a separate issue on this that would be handled separately than in this pr. 

I do see your point though on it being confusing.